content/en/agent/troubleshooting/debug_mode.md

```diff
@@ -79,10 +104,21 @@ Or the container can be restarted.
 {{% /tab %}}
 {{< /tabs >}}
 
+## Agent log level
+
+The following Agent log levels are available for `log_level` or `DD_LOG_LEVEL`:
+
+| Option  | Fatal logs | Error logs | Warn logs | Info logs | Debug logs | Trace logs | All logs |
+|---------|------------|------------|-----------|-----------|------------|------------|----------|
+| `OFF`   |            |            |           |           |            |            |          |
+| `FATAL` | X          |            |           |           |            |            |          |
```

Comment:
Nice, do you think we could use an icon like: ✅ ? Like we have here: https://docs.datadoghq.com/integrations/amazon_lambda/#overview

Reply:
These icons actually look different on Windows machines. I think if we are going to do something like this, we should have brand design make us an icon. Maybe something similar using the datadog purple color?